src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -109,6 +117,16 @@ public function queryAction(Request $request)
         );
     }
 
+    private function resolveIndexPlaceholders($value)
+    {
+        $resolver = $this->container->get('twig');
```

Comment:
Using twig here feels like an unnecessary overhead to me... Why not simply using `str_replace`?